app/code/Magento/MediaGalleryApi/Model/Asset/Command/SaveInterface.php

```diff
@@ -12,14 +12,14 @@
 
 /**
  * A command which executes the media gallery asset save operation.
+ * @api
  */
 interface SaveInterface
 {
     /**
      * Save media asset
      *
      * @param \Magento\MediaGalleryApi\Api\Data\AssetInterface $mediaAsset
-     *
```

Comment:
What is the return result? What `int` means?

Reply:
It's the media asset id, comment for the method was updated